controllers/offer.js

```diff
@@ -0,0 +1,60 @@
+const offerService = require('~/services/offer')
+
+const getOffers = async (req, res) => {
+  const offers = await offerService.getOffers()
+
+  res.status(200).json(offers)
+}
+
+const getOfferById = async (req, res) => {
+  const { id } = req.params
+
+  const offer = await offerService.getOfferById(id)
+
+  res.status(200).json(offer)
+}
+
+const createOffer = async (req, res) => {
+  const userId = req.params.id
```

Comment:
maybe better to use destructuring:
const { id: userId } = req.params

Reply:
const { id } = req.params

Reply:
refactored